.env.example

```diff
@@ -0,0 +1,4 @@
+# Environment Variables used for Cypress e2e testing
+CYPRESS_REMOTE_DEBUGGING_PORT=9222
+NETWORK_NAME=goerli
+SECRET_WORDS='test test test test test test test test test test test junk'
```

Comment:
nit: can we change the name to `TEST_WALLET_SEED` or similar? Just to be a bit more clear as to what it is

Reply:
Unfortunately I don't think we can change the variable names here - if changed Synpress doesn't pick get the values. There might be a way to change this in the config but did some digging and haven't been successful

Reply:
All good!